src/composables/auth.ts

```diff
@@ -96,12 +92,12 @@ export const useAuth = createCustomScopedComposable(() => {
 
   async function logout() {
     if (IS_EXTENSION) {
-      const sessionEncryptionData = await getSessionEncryptionData();
-      if (sessionEncryptionData) {
+      const sessionEncryptionKey = await getSessionEncryptionKey();
+      if (sessionEncryptionKey) {
```

Comment:
Why we need to check if the session key exists? I think logging out means resetting the key and the authenticated state. Or I'm missing something here?